zgw_consumers/admin_fields.py

```diff
@@ -45,21 +48,40 @@ def get_zaaktypen() -> Dict[Service, List[Dict[str, Any]]]:
 
 
 def get_zaaktype_field(db_field: Field, request: HttpRequest, **kwargs):
-    zaaktypen = get_zaaktypen()
-
-    def _get_choice(zaaktype: dict) -> Tuple[str, str]:
-        return (
-            zaaktype["url"],
-            f"{zaaktype['identificatie']} - {zaaktype['omschrijving']}",
+    try:
+        zaaktypen = get_zaaktypen()
+    except ClientError as exc:
+        error_json = exc.args[0]
+        error_cause_json = exc.__cause__.args[0]
```

Comment:
this is not correct, since `exc.__cause__` is supposed to be a `requests.RequestsException` instance.

And what that contains, you can see here: https://requests.readthedocs.io/en/latest/_modules/requests/exceptions/#RequestException